src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,532 @@
+import { useAppSelector } from '../../../../store/hooks';
+import { useGetOscapCustomizationsQuery } from '../../../../store/imageBuilderApi';
+import { useGetSourceListQuery } from '../../../../store/provisioningApi';
+import { useShowActivationKeyQuery } from '../../../../store/rhsmApi';
+import {
+  selectActivationKey,
+  selectArchitecture,
+  selectAwsAccountId,
+  selectAwsShareMethod,
+  selectBlueprintDescription,
+  selectBlueprintName,
+  selectCustomRepositories,
+  selectDistribution,
+  selectGcpAccountType,
+  selectGcpEmail,
+  selectGcpShareMethod,
+  selectProfile,
+  selectRegistrationType,
+} from '../../../../store/wizardSlice';
+import { useGetEnvironment } from '../../../../Utilities/useGetEnvironment';
+
+const ExpirationWarning = () => {
+  return (
+    <div className="pf-u-mr-sm pf-u-font-size-sm pf-u-warning-color-100">
+      <ExclamationTriangleIcon /> Expires 14 days after creation
+    </div>
+  );
+};
+
+export const ImageOutputList = () => {
+  const distribution = useAppSelector((state) => selectDistribution(state));
+  const arch = useAppSelector((state) => selectArchitecture(state));
+  return (
+    <TextContent>
+      <ReleaseLifecycle />
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Release
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {RELEASES.get(distribution)}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Architecture
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>{arch}</TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+export const FSCList = () => {
+  return (
+    <TextContent>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvAWSList = () => {
+  const { data: rawAWSSources, isSuccess } = useGetSourceListQuery({
+    provider: 'aws',
+  });
+  const awsAccountId = useAppSelector((state) => selectAwsAccountId(state));
+  const awsShareMethod = useAppSelector((state) => selectAwsShareMethod(state));
+
+  const awsSources = extractProvisioningList(rawAWSSources);
+  const { isBeta } = useGetEnvironment();
+
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>AWS</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Shared to account
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {!isBeta() && awsAccountId}
```

Comment:
I think that now that Provisioning is GA we can get rid of the `isBeta()` checks... so:
```
        <TextListItem component={TextListItemVariants.dd}>
          {awsShareMethod === 'sources' && isSuccess && awsAccountId}
          {awsShareMethod === 'manual' && awsAccountId}
        </TextListItem>
```

Reply:
Let's do a follow-up PR.